style.css

```diff
@@ -97,6 +150,307 @@ h2 {
+  align-items: center;
+  gap: 12px;
+}
+
+.works .work-items .work-item .work-info .work-info-details .company-name {
+  font-weight: 500;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+}
+
+.works .work-items .work-item .work-info .work-info-details .position {
+  font-weight: 500;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.works .work-items .work-item .work-info .work-info-details .date {
+  font-weight: 500;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.works .work-items .work-item .work-info .project-description {
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  margin-top: 20px;
+  margin-bottom: 12px;
+}
+
+.works .work-items .work-item .work-info .project-technologies {
+  display: -webkit-box;
+  display: flex;
+  gap: 8px;
+  margin-bottom: 24px;
+  list-style: none;
+}
+
+.works .work-items .work-item .work-info .project-technologies li {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  padding: 4px 12px;
+  color: #6070ff;
+  background-color: #ebebff;
+  border-radius: 8px;
+}
+
+.works .work-items .work-item .work-info .see-project {
+  text-decoration: none;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  display: inline-block;
+  color: #396df2;
+  padding: 12px;
+  border: 1px solid #396df2;
+  border-radius: 8px;
+}
+
+.works .work-items .work-item .work-info .see-project:hover {
+  background-color: #396df2;
+  color: #fff;
+  -webkit-transition: 0.3s;
+  transition: 0.3s;
+}
+
+@media screen and (min-width: 767.98px) {
+  header nav ul {
+    list-style: none;
+    display: flex;
+    align-items: center;
+  }
+
+  header nav ul li a {
+    padding: 10px 12px;
+    color: #344563;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    text-decoration: none;
+  }
+
+  header nav .menu-btn {
+    display: none;
+  }
+
+  .works {
+    margin-top: 72px;
+  }
+
+  .works .work-items {
+    row-gap: 82px;
+  }
+
+  .works .work-items .work-item {
+    max-width: 100%;
+    margin: auto;
+    border-radius: 24px;
+    padding: 12px;
+    gap: 12px;
+  }
+
+  .works .work-items .work-item:nth-child(odd) {
+    padding-right: 0;
+    flex-direction: row;
+  }
+
+  .works .work-items .work-item .work-info {
+    width: 50%;
+    padding: 0 12px;
+  }
+
+  .works .work-items .work-item:nth-child(odd) .work-info {
+    padding-right: 12px;
+  }
+
+  .works .work-items .work-item:nth-child(even) {
+    padding-left: 0;
+    flex-direction: row-reverse;
+  }
+
+  .works .work-items .work-item:nth-child(even) .work-info {
+    padding-left: 24px;
+  }
+
+  .works .work-items .work-item .work-img {
+    width: 50%;
+    padding: 0;
+  }
+}
+
+@media screen and (min-width: 1101.98px) {
+  .banner {
+    background-size: contain;
+    background-position: center;
+    padding: 200px 24px 170px 24px;
+    border-bottom-left-radius: 80px;
+  }
```

Comment:
- [ ] The background image is missing from both mobile and desktop version. Also give some margin between paragraph and `let's connect`.

### Your Design.

![Screenshot from 2021-06-11 16-55-29](https://user-images.githubusercontent.com/22434660/121679342-de382b80-cad5-11eb-87dd-cc424a5c40f3.png)

### Figma design

![Screenshot from 2021-06-11 16-57-53](https://user-images.githubusercontent.com/22434660/121679652-32dba680-cad6-11eb-8a18-38eb8cdf1fc0.png)